spec/channels/conversation_channel_spec.rb

```diff
@@ -0,0 +1,180 @@
+require 'rails_helper'
+
+describe ConversationChannel, type: :channel do
+  let(:conversation) { create(:conversation) }
+  let(:params) do
+    {
+      conversation_id: conversation.id
+    }
+  end
+  let(:user)            { create(:user) }
+  let(:message)         { 'Hello, Rails!' }
+  let(:channel)         { "Conversation-#{params[:conversation_id]}" }
+
+  describe 'with valid params' do
+    let(:data) do
+      {
+        sender_id: user.id,
+        conversation_id: conversation.id,
+        message: message
+      }
+    end
+
+    context 'subscribe to existing conversation' do
```

Comment:
If it's possible, the context should start with where or when, in this case: `when subscribing to existing conversations`